.config/nextest.toml

```diff
@@ -0,0 +1,10 @@
+[test-groups]
+serial-integration = { max-threads = 1 }
+
+#[[profile.default.overrides]]
+#filter = 'package(mc-eth)'
+#test-group = 'serial-integration'
+#
+#[[profile.default.overrides]]
+#filter = 'package(mc-e2e-tests)'
+#test-group = 'serial-integration'
```

Comment:
Can this be removed?

Reply:
I thought be helpful to leave in case you wanted to run sequentially- but happy to kill.

Thoughts?

Reply:
I would document it, then